Sources/Logging/Docs.docc/BestPractices/003-AcceptingLoggers.md

```diff
@@ -0,0 +1,100 @@
+# 003: Accepting loggers in libraries
+
+Accept loggers through method parameters to ensure proper metadata propagation.
```

Comment:
Would you consider splitting this one into a separate PR to allow us to discuss more, without blocking the other changes? I have more thoughts here.